src/realmCryo.ttl

```diff
@@ -234,24 +431,59 @@ soreac:SeaIce rdf:type owl:Class ;
+                 skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_03000071> ;
+                 rdfs:comment "Sea ice develops from young ice; thickness from 0.3 to 2 meters (1 to 6.6 feet).  Sea ice be subdivided into thin first-year ice (white ice), medium first-year ice, and thick first-year ice.  First-year ice is distinguished from older ice primarily by having a higher salinity. Undeformed first-year ice differs from older ice in that it is smoother and lacks refrozen melt ponds. Characteristically level where undisturbed by pressure, but where ridges occur, they  distinguished by being larger, more angular, and more porous than multiyear ridges."@en ;
+                 skos:definition  [
+                   rdfs:comment  "Sea ice which has formed over a single freezing season."@en ;
+                   dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+                   dcterms:created "2020-02-12T13:45:28-08:00Z"^^xsd:dateTimeStamp ;
+                   dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                   prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_03000071> ;
+                 ] .
 
 
 ###  http://sweetontology.net/realmCryo/Snout
 soreac:Snout rdf:type owl:Class ;
-           rdfs:label "snout"@en .
+           rdfs:label "snout"@en ;
+           skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001529> ;
+           skos:definition  [
+             rdfs:comment  "An ice mass which constitutes that part of a glacier which has the lowest elevation."@en ;
+             dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+             dcterms:created "2020-02-12T13:48:09-08:00Z"^^xsd:dateTimeStamp ;
+             dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+             prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01001529> ;
+           ] .
 
 
 ###  http://sweetontology.net/realmCryo/TabularIceberg
 soreac:TabularIceberg rdf:type owl:Class ;
                     rdfs:subClassOf soreac:Iceberg ;
-                    rdfs:label "tabular iceberg"@en .
+                    rdfs:label "tabular iceberg"@en ;
+                    skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001534> ;
+                    rdfs:comment "Newly formed tabular icebergs have nearly vertical sides and flat tops. In the Antarctic, they may be tens of kilometers wide, up to 160 km (100 miles) long, and as much as 300 m (1000 ft) thick, with about 30 m (100 ft) exposed above the sea surface. In the Arctic, large icebergs of this type are called ice islands, but they are considerably smaller than the largest of the antarctic variety. Has synonyms tabular berg, table iceberg. Formerly called barrier iceberg."@en ;
+                    skos:definition  [
+                      rdfs:comment  "An iceberg which 1) has a flat upper surface, 2) is derived from an ice shelf, ice tongue, or floating tidewater glacier via detachment."@en ;
```

Comment:
is the origin really a necessary or sufficient property, or just a note?  Seems like if the term is 'tabular iceberg' its shape would constitute the necessary and sufficient properties, and its origin would be interesting but not a defining property.